embed/src/main/java/orca/embed/cloudembed/controller/InterDomainHandler.java

```diff
@@ -957,6 +957,7 @@ public void createManifest(NetworkElement requestElement,OntModel manifestModel,
                 boolean isStitchingDomain = NdlCommons.isStitchingNodeInManifest(start.getResource());
                 link_url = start.getURI();
                 if(isStitchingDomain){
+                	link_url = start.getName();
```

Comment:
It looks like the `source domain` and `destination domain` code here and below should be refactored to reduce code duplication.